app/src/main/res/layout/revision_card_activity.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:tools="http://schemas.android.com/tools"
-  android:id="@+id/review_card_fragment_placeholder"
+  android:id="@+id/revision_card_fragment_placeholder"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
-  tools:context="ReviewCardActivity" />
+  tools:context="Re" />
```

Comment:
Check this you are missing something here.

Reply:
Done